[17.0][ADD] partner_contact_telegram: New module

AlvaroRM11 opened the pull request:
@BinhexTeam

The purpose of this PR is to add a field to enter the Telegram nickname for contacts. 
Depends on #2054

christian-ramos-tecnativa commented on partner_contact_telegram/models/res_partner.py:
```diff
@@ -0,0 +1,57 @@
+
+    _inherit = "res.partner"
+
+    telegram_id = fields.Char()
```
This doesn't seem to be used and is hidden, is it really needed?

pedrobaeza replied:
Suffix `_id` is reserved for many2one fields.

arielbarreiros96 replied:
Indeed using `_id` could be confusing for a char field.
In Telegram there is also has a unique identifier for users who don't share phone number or alias for privacy reasons. While Telegram users don't have access to this identifier, the Telegram API does provide this information, and it could be helpful for retrieving users using `search()` for example.

AlvaroRM11 committed:
[ADD] partner_contact_telegram: New module

[IMP] partner_contact_telegram: Support partner_contact_social_media_information_page module

AlvaroRM11 committed:
[DON'T MERGE] test-requirements.txt
